index.html

```diff
@@ -25,10 +25,10 @@
 	  {% else %}{% assign author = site.authors["default"] %}{% endif %}
       <div class="small-author">
         <span class="name">{{author.display_name}}</span>
-        {% if author.gravatar %}<img src="http://www.gravatar.com/avatar/{{ author.gravatar }}?s=28">
+        {% if author.gravatar %}<img src="https://www.gravatar.com/avatar/{{ author.gravatar }}?s=28">
```

Comment:
Would it be better to use `<img src="//www.gravatar.com/avatar/{{ author.gravatar }}?s=28">`? Then the image loads over whatever protocol the page is being served over.